app/models/user/User.scala

```diff
@@ -168,11 +225,77 @@ class UserDAO @Inject()(sqlClient: SqlClient)(implicit ec: ExecutionContext)
                          requestingUser: User)(implicit ctx: DBAccessContext): Fox[List[User]] =
     for {
 
-      selectionPredicates <- buildSelectionPredicates(isEditable, isTeamManagerOrAdmin, isAdmin, requestingUser)
+      selectionPredicates <- buildSelectionPredicates(isEditable,
+                                                      isTeamManagerOrAdmin,
+                                                      isAdmin,
+                                                      requestingUser,
+                                                      SqlToken.raw(""))
       r <- run(q"select $columns from $existingCollectionName where $selectionPredicates".as[UsersRow])
       parsed <- parseAll(r)
     } yield parsed
 
+  implicit def GetResultUserCompactInfo(implicit e0: GetResult[String],
+                                        e1: GetResult[java.sql.Timestamp],
+                                        e2: GetResult[Boolean],
+                                        e3: GetResult[Option[String]]): GetResult[UserCompactInfo] = GetResult { prs =>
+    import prs._
+    // format: off
+    UserCompactInfo(<<[String],<<[String],<<[String],<<[String],<<[String],<<[String],<<[Boolean],<<[Boolean],
+      <<[Boolean],<<[Boolean],<<[String],<<[String],<<[String],<<[String], <<[String],<<[java.sql.Timestamp],<<[String],
+      <<[String],<<[String],<<[String],<<[java.sql.Timestamp],<<?[String],<<[Boolean],<<[Boolean]
+    )
```

Comment:
Hmm, since this variant also needs to list all the data types, it does not appear to me to bring that much benefit. So I’d vote to keep it unified and use what we had before. Do you agree?

Reply:
Unfortunately, I can not just use the one from e.g. listing Annotations, because in Scala tuples can only have 22 elements (here 24 are needed) and thus the row can not be parsed as a tuple. :(

Reply:
Ah well ok then let’s keep your variant :D